Bottom: 54be3fc99b5df31fdb21c3393b63dd885914a2bc
Top:    64ba82243c1c97f061de371b1549f2eef22604f6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 13:18:26 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index f1853a3..cf38e16 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -23,11 +23,7 @@
 #ifndef __HIDGL_INCLUDED__
 #define __HIDGL_INCLUDED__
 
-//#define TRIANGLE_ARRAY_SIZE 5000
 #define TRIANGLE_ARRAY_SIZE 5461
-/* Assumes GLFloat is 4 bytes, and we have X,Y(,Z) coords x3 for each triangle:
-   4 * 5461 * 2 * 3 = 109464 */
-//#define TRIANGLE_ARRAY_BYTES 131072
 typedef struct {
   GLfloat triangle_array [3 * 3 * TRIANGLE_ARRAY_SIZE];
   unsigned int triangle_count;
